test/Mono.Linker.Tests.Cases/Reflection/TypeHierarchyReflectionWarnings.cs

```diff
@@ -146,9 +166,13 @@ class AnnotatedPublicProperties
 			[KeptAttributeAttribute (typeof (DynamicallyAccessedMembersAttribute))]
 			[DynamicallyAccessedMembers (DynamicallyAccessedMemberTypes.PublicMethods)]
 			public static string DAMProperty {
-				// Property access reports warnings on getter/setter
-				[ExpectedWarning ("IL2114", nameof (AnnotatedPublicProperties), nameof (DAMProperty) + ".get")]
+				[Kept]
+				// No warning for getter since return value is not annotated
```

Comment:
The return value is annotated, but because the getter is not virtual it doesn't matter. If the property was virtual this should/would warn.